### PR TITLE
refactor: Moved qualification to cozy-client

### LINKED ISSUE #1042: Moved the locales and the Qualification themes in cozy-client
Suite à cette PR: https://github.com/cozy/cozy-client/pull/1037

Aujourd'hui `cozy-client` porte déjà les qualifications:
- https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/models/document.js
- https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/assets/qualifications.json

Il faudrait rapatrier:
- Les catégories:
  - https://github.com/cozy/cozy-libs/blob/master/packages/cozy-scanner/src/DocumentTypeData.js
  - https://github.com/cozy/cozy-libs/blob/master/packages/cozy-scanner/src/DocumentTypeDataHelpers.js

- Les locales:
  - https://github.com/cozy/cozy-libs/blob/master/packages/cozy-scanner/src/locales/fr.json#L19
  - https://github.com/cozy/cozy-libs/blob/master/packages/cozy-scanner/src/locales/en.json#L19
  
Je pense que c'est à peu près tout, hésitez pas à compléter si nécessaire 🙂

### PR DESCRIPTION
**Pull request by Merkur39**
Moved everything related to qualification from the cozy-scanner lib to document models in cozy-client
fix #1042

### CODE REVIEW

**Comment by Merkur39** [comment]
[Retours tech](https://trello.com/c/plGHnR1V/736-retours-tech)

File: packages/cozy-client/src/models/document/locales/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Polyglot from 'node-polyglot'
```
Comment (paultranvan) [inline packages/cozy-client/src/models/document/locales/index.js:1]: This should be added to the package.json

File: packages/cozy-client/src/models/document/DocumentTypeData.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
+import { Qualification } from './qualification'
```
Comment (paultranvan) [inline packages/cozy-client/src/models/document/DocumentTypeData.js:1]: I find it weird to have `DocumentTypeData`, `DocumentTypeDataHelpers` with the first letter in uppercase, while `qualification.js` is lowercase. I think the overall style is to have capital letters when the default export is a class, so we might want to have `documentTypeData` and `documentTypeDataHelpers` ?

File: packages/cozy-client/src/models/document/locales/index.js (hunk 1 above)
Comment (Merkur39) [reply packages/cozy-client/src/models/document/locales/index.js:1]: Oops, good point

File: packages/cozy-client/src/models/document/DocumentTypeData.js (hunk 2 above)
Comment (Merkur39) [reply packages/cozy-client/src/models/document/DocumentTypeData.js:1]: Indeed, it makes more sense

**Review by paultranvan — APPROVED** [review]
Looks good, except for one nit:  `DocumentTypeData.d.ts`,  `DocumentTypeDataHelpers.d.ts` and `Qualification.d.ts` should be renamed without the first uppercase letter ;)

**Review by Crash-- — APPROVED** [review]

**Comment by Crash--** [comment]
ping @Merkur39

**Pull request merged**